src/libraries/System.Security.Claims/src/System/Security/Claims/ClaimsIdentity.cs

```diff
@@ -910,26 +909,26 @@ protected virtual void WriteTo(BinaryWriter writer, byte[] userData)
 
             if ((mask & SerializationMask.HasLabel) == SerializationMask.HasLabel)
             {
-                writer.Write(_label);
+                writer.Write(_label!);
             }
 
             if ((mask & SerializationMask.HasClaims) == SerializationMask.HasClaims)
             {
                 writer.Write(_instanceClaims.Count);
                 foreach (var claim in _instanceClaims)
                 {
-                    claim.WriteTo(writer);
+                    claim!.WriteTo(writer);
```

Comment:
What's guaranteeing this is non-null?

Reply:
it was extra bang, i was marked instanceClaims list can have null, because many access checking for nulls, then reilized non null value added to the list, so now it is non nullable list of instanceClaims, removed.